cutter-plugin/R2GhidraDecompiler.cpp

```diff
@@ -16,51 +17,6 @@ R2GhidraDecompiler::R2GhidraDecompiler(QObject *parent)
 
 void R2GhidraDecompiler::decompileAt(RVA addr)
```

Comment:
`addr` is ignored.

Reply:
I have noticed this already. But the decompileAt() in R2Dec needs this, that's why I didn't change it here. If the core can be used in R2Dec::decompileAt(), we can avoid this here. This has to be decided.

Reply:
Looks good. Weird that I can't mark this comment as resolved.

Reply:
I will mark it as resolved:)